src/components/learner-credit-management/data/utils.js

```diff
@@ -95,6 +95,16 @@ export const transformUtilizationTableResults = results => results.map(result =>
   courseKey: result.courseKey,
 }));
 
+export const transformUtilizationTableTransactionResults = results => results.map(result => ({
```

Comment:
nit: `transformUtilizationTableSubsidyTransactionResults` just to be a bit more explicit compared to "offer transactions/redemptions" (reduce any naming ambiguity).